hibernate-core/src/main/java/org/hibernate/event/internal/DefaultLoadEventListener.java

```diff
@@ -673,11 +673,11 @@ private Object loadFromSecondLevelCache(
 			final EntityKey entityKey) {
 
 		final SessionImplementor source = event.getSession();
-		final boolean useCache = persister.canReadFromCache()
-				&& source.getCacheMode().isGetEnabled()
-				&& event.getLockMode().lessThan( LockMode.READ );
+		final boolean skipCache = ( ! persister.canReadFromCache() )
+				|| ( ! source.getCacheMode().isGetEnabled() )
+				|| ( ! event.getLockMode().lessThan( LockMode.READ ) );
```

Comment:
I must be missing something; I don't understand how the above change would make any difference performance-wise.

Was this change made to be more readable, rather than to improve performance?

Reply:
you're right. This was meant to have some additional changes but I then changed them back.  I'll remove it as it became unnecessary noise. Thanks!